rasa/core/domain.py

```diff
@@ -264,31 +264,68 @@ def collect_slots(slot_dict: Dict[Text, Any]) -> List[Slot]:
         return slots
 
     @staticmethod
+    def _transform_intent_properties_for_internal_use(
+        intent: Dict[Text, Any], entities: List
+    ) -> Dict[Text, Any]:
+        """Transform the intent properties `use_entities` and `ignore_entities` that are
+        used in domain files into one internal property `used_entities`. Use default
+        values where the intent properties are not explicitly provided.
+        """
+        name, properties = list(intent.items())[0]
+
+        properties.setdefault("use_entities", True)
+        properties.setdefault("ignore_entities", [])
+        if properties["use_entities"] is None or properties["use_entities"] is False:
```

Comment:
can be shortened to:
```suggestion
        if not properties["use_entities"]:
```

Reply:
in any case, a comment would be helpful

Reply:
that doesn't really matter right? Worst case is that an empty list is replaced with an empty list

Reply:
true 👍